rooms/views.py

```diff
@@ -148,4 +148,73 @@ def get(self, request):
 
             return JsonResponse({'rooms' : result}, status=200)
         except IndexError:
-            return JsonResponse({'result' : 'INVALID GEOCODE'}, status=400)
\ No newline at end of file
+            return JsonResponse({'result' : 'INVALID GEOCODE'}, status=400)
+
+class RoomDetailView(View):
+    # @UserTokenDeco
+    def get(self, request, room_id):
+        try:
+            room               = Room.objects.select_related('host').prefetch_related('image_set', 'host__socialflatform_set', 'roomoptioninfo_set', 'roomconvenience_set').get(id=room_id)
+            user               = 2
+            # user              = request.user
+            wish               = True if Wish.objects.filter(user_id = user, room_id = room).exists()  else False
+            image_path         = [images.s3_url for images in room.image_set.all()]
```

Comment:
단수와 복수는 명확하게 구분해서 사용해야 합니다!
`room.image_set.all()`은 list를 반환하고, 그 안에 들어있는 image 인스턴스 하나 하나를 반복문을 통해서 꺼내오고 있기 때문이겠죠?

```suggestion
            image_paths         = [image.s3_url for image in room.image_set.all()]
```

Reply:
수정완료했습니다 :)